README.md

```diff
@@ -47,6 +47,10 @@ pip install cognite-pygen
 ```bash
 pip install cognite-pygen[cli]
 ```
+If using zsh:
+```bash
+pip install 'cognite-pygen[cli]'
```

Comment:
IIRC bash will also let you use quotes around a command or option to denote it, so `pip install "cognite-pygen[cli]"` should work for both (all?) available *NIX shells.